cpp/src/slice2matlab/Main.cpp

```diff
@@ -45,7 +45,7 @@ namespace
 
         out << nl << "% Copyright (c) ZeroC, Inc.";
         out << nl << "% Generated from " << f << " by slice2matlab version " << ICE_STRING_VERSION;
```

Comment:
is the missing new line between the two intentional? If so we should add a comment to clarify it.

Reply:
MATLAB is different from the other compilers, with the copyright notice not at the top of the file. I didn't change it in this PR. We should review it when we review the MATLAB doc-comment generation.